en/nix-modules.md

```diff
@@ -0,0 +1,152 @@
+---
+order: 3
+---
+
+# Introduction to Nix modules
+
+Learning to work with the [[modules|module system]] in [[nixpkgs|nixpkgs]] is a big stepping stone to writing easy to maintain and shareable Nix code. We will focus on that in this tutorial -- by understanding how to use `evalModules` from [[nixpkgs|nixpkgs]] to define and use our own modules. The next tutorial in this series will talk about how to do that using `flake-parts` for writing [[flakes|flakes]] and sharing the modules with others flakes. 
+
+>[!tip] Everything about the module system
+> This tutorial introduces the reader to the Nix module system. To learn more about the module system, we recommend [this video from Tweag](https://www.youtube.com/watch?v=N7hFP_40DJo) as well the article "[Module system deep dive][doc]" from nix.dev.
+
+[doc]: https://nix.dev/tutorials/module-system/module-system.html
```

Comment:
This should explain
- what we'll do in this tutorial
- why we do it
- outline how we do it; see other comment